README.md

```diff
@@ -1716,6 +1716,13 @@ Thanks goes to these wonderful people ([:hugs:](https://allcontributors.org/docs
                     <sub><b>Kaone Keboetseng</b></sub>
                 </a>
             </td>
+            <td align="center">
+                <a href="https://arshadmq.com/">
+                    <img src="https://avatars.githubusercontent.com/u/54259677?v=4" width="100px;" alt="Ali Solanki"/>
```

Comment:
```suggestion
                    <img src="https://avatars.githubusercontent.com/u/54259677?v=4" width="100px;" alt="Mohd Arshad"/>
```

Reply:
Thanks for pointing that out, its been resolved